src/html/addNewRecipeExtTml.xhtml

```diff
@@ -1,18 +1,19 @@
-<h3 class="text-center"><strong>Add New Recipe</strong></h3>
+<h3 class="text-center">
+    <strong>Add New Recipe</strong>
+</h3>
 <div class="form-group">
     <label>Recipe title:</label>
-    <input class="form-control"  data-ng-model="recipeLink" placeholder="Title..." />
+    <input class="form-control" data-ng-model="recipeLink" placeholder="Title..."/>
 </div>
 <!-- /.form-group -->
 <div class="form-group">
     <label>Recipe description:</label>
-    <textarea class="form-control" rows="3" data-ng-model="recipeDesc"> </textarea>
+    <textarea class="form-control" rows="3" data-ng-model="recipeDesc"></textarea>
 </div>
 <!-- /.form-group -->
 <div class="form-group">
     <label>Recipe image:</label>
-    <input type="file" data-ng-model="recipeImg" />
+    <input type="file" data-ng-model="recipeImg"/>
 </div>
 <!-- /.form-group -->
-<button type="button" class="btn btn-lg btn-primary" data-ng-controller="dialogController" data-ng-click="closePopup()"
-data-ng-disabled="!recipeTitle.length &&!recipeDesc.length">Add</button>
+<button type="button" class="btn btn-lg btn-primary" data-ng-click="closePopup()" data-ng-disabled="!recipeTitle.length &&!recipeDesc.length">Add</button>
```

Comment:
It's better to make a function than this type of logic in view.

"data-ng-disabled="!recipeTitle.length &&!recipeDesc.length""